lib/router.js

```diff
@@ -45,18 +46,26 @@ router.get('/slashCommands/ci', (req, res) => {
 router.get('/slashCommands/ext', (req, res) => {
     const command = req.query && req.query.command;
     const text = req.query.text;
+    const update = req.query.update;
```

Comment:
We could still easily (somewhat) accomplish this by `.split()`ing `text` by space, then checking the last popped match for `update`. So it would be the same code for `/ext update` or `/ext john update`... Your call.